examples/run-blockchain/test-data.json

```diff
@@ -0,0 +1,159 @@
+{
+  "blocks": [
+    {
+      "blockHeader": {
+        "bloom": "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
if we make sure that all the hex values are prepended with '0x' we can drop a bunch the formating stuff